File: File-Export-Service/src/main/java/com/kinok0/fileexportservice/repository/TaskRepository.java
package com.kinok0.fileexportservice.repository;

import com.kinok0.fileexportservice.entity.TaskEntity;
import com.kinok0.fileexportservice.entity.UserEntity;
import org.springframework.data.repository.CrudRepository;

import java.util.List;
import java.util.UUID;

public interface TaskRepository extends CrudRepository<TaskEntity, UUID> {

    @Override
    List<TaskEntity> findAll();
}

File: Kanban-Service/src/main/kotlin/com/kinok0/kanbanservice/repository/TaskRepository.kt
package com.kinok0.kanbanservice.repository

import com.kinok0.kanbanservice.entity.TaskEntity
import com.kinok0.kanbanservice.entity.TaskStatement
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface TaskRepository : CrudRepository<TaskEntity, UUID> {

    fun findTaskEntityById(id: UUID) : TaskEntity

    fun findTaskEntitiesByStatement(statement: TaskStatement) : MutableList<TaskEntity>

    fun deleteTaskEntityById(id: UUID)

}